### PR TITLE
Add Stats to cloud.jetpack.com/pricing

### PR DESCRIPTION
**Pull request by IanRamosC**
Context: p1HpG7-nWn-p2#comment-65153

## Proposed Changes

* Update the pricing page to include Stats (not only on the post-connection pricing page)
* Update the lightbox info for Stats:
  * Add benefits included
  * Add features included
  * Add "Great for" content
* Update description, short description, and display names
* Remove hardcoded pricing

image

## Testing Instructions

* Open Jetpack Cloud live link
* Visit the pricing page `/pricing`
* Confirm that Stats is present and matches copy in the document linked above
* Test that pricing is correct and that you are able to complete the purchase of the product
* Extra: Open Calypso live link and `visit /stats/purchase`, select a site and play with going to the cart through the decision path for Stats. Make sure it redirects the same way it's currently doing on the production code

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Review by robertsreberski — APPROVED** [review]
Good code and tests well :shipit:

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8717857

Thank you @IanRamosC for including a screenshot in the description! This is really helpful for our translators.

File: packages/calypso-products/src/translations.tsx [hunk 1]
```diff
@@ -1200,6 +1229,17 @@ export const getJetpackProductsBenefits = (): Record< string, Array< TranslateRe
 		translate( 'Repurpose, reuse or republish already published content' ),
 	];
 
+	const statsCommercialBenefits = [
+		translate( 'Better understand your audience' ),
+		translate( 'Discover your top performing posts & pages' ),
+		translate( 'Get detailed insights on the referrers that bring traffic from your site' ),
```
Comment (emilyaudela) [inline packages/calypso-products/src/translations.tsx:1235]: Question from our translation team. Should this be "bring traffic to your site" not "from" your site?

File: packages/calypso-products/src/translations.tsx [hunk 2]
```diff
@@ -979,6 +997,15 @@ export const getJetpackProductsWhatIsIncluded = (): Record< string, Array< Trans
 		translate( 'Recycle content' ),
 		translate( 'Image generator' ),
 	];
+	const statsCommercialIncludesInfo = [
+		translate( 'Real-time data on visitors' ),
+		translate( 'Traffic stats and trends for post and pages' ),
```
Comment (emilyaudela) [inline packages/calypso-products/src/translations.tsx:1002]: "post" should probably be "posts" (plural) here, right?

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8717857).

**Pull request merged**